packages/jest-expo/src/preset/setup.js

```diff
@@ -171,6 +173,10 @@ try {
     //
     // NOTE: The adapter validates the number of arguments, which we don't do in the mocked functions.
     // This means the mock functions will not throw validation errors the way they would in an app.
+
+    ExpoModulesCore.uuid.v4 = uuid.uuid.v4;
+    ExpoModulesCore.uuid.v5 = uuid.uuid.v5;
```

Comment:
```suggestion
    // Mock the `uuid` object with the implementation for web.
    ExpoModulesCore.uuid = uuid;
```

Reply:
This isn't working for me as it leads to `TypeError: Cannot set property uuid of [object Object] which has only a getter` when running the tests. That's why I reassigned properties one-by-one